README.md

```diff
@@ -52,6 +53,28 @@ To start developing navigate to your theme's folder (`cd theme_name`) and run:
 npm start
 ```
 
+## :tada: Awesome features
+
+Here's why you should use Eightshift Boilerplate for your next theme.
+
+- **OOP**
+  - All code is written following the latest object-oriented programming paradigms
+  - Written using PSR-4, (denis?)
```

Comment:
Nope 😄For theme to be `PSR-4` autoloading we'd need to change how files are named, and ideally use `PSR-2`standards.

For now we are still using `classmap` for autoloading 🙂